app/models/recipe.rb

```diff
@@ -25,6 +25,7 @@ class Recipe < ActiveRecord::Base
   validates_presence_of :name, :directions, :category, :author, :message => _("Can't be blank")
   validates_presence_of :photo_file_name, :message => _("It is imperative that you put a really nice one ;)")
   validates_length_of :name, :minimum => 5
+# validates_attachment :photo_file_name, content_type: { content_type: ["image/jpg", "image/jpeg", "image/png", "image/gif"] }
```

Comment:
Line is too long. [126/80]<br>Incorrect indentation detected (column 0 instead of 2).